values/gitea/gitea.gotmpl

```diff
@@ -81,10 +81,10 @@ gitea:
       ENABLE_OPENID_SIGNIN: true
       ENABLE_OPENID_SIGNUP: true
     oauth2_client:
-      OPENID_CONNECT_SCOPES: openid email profile
       ENABLE_AUTO_REGISTRATION: true
-      ACCOUNT_LINKING: auto
-      USERNAME: email
+      ACCOUNT_LINKING: disabled
+      OPENID_CONNECT_SCOPES: openid email profile
+      USERNAME: nickname
```

Comment:
Would it be possible to use the `username` scope and to set compatible username in apl-tasks: github/linode/apl-tasks/blob/pr-3.5.0/src/tasks/keycloak/config.ts#L73?